packages/yugabyte-client-package-script/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e -x
+if [ -d "/opt/yugabyte/" ]; then
+  if [ -L "/usr/bin/cqlsh" ] && [ -L "/usr/bin/ysqlsh" ]; then
+    rm -f /usr/bin/ysqlsh
+    rm -f /usr/bin/cqlsh
+    cd /opt/yugabyte-client/ || exit 1
+    ./bin/post_install.sh
+    ln -s /opt/yugabyte-client/bin/cqlsh /usr/bin/cqlsh
+    ln -s /opt/yugabyte-client/postgres/bin/ysqlsh /usr/bin/ysqlsh
+  else
+    cd /opt/yugabyte-client/ || exit 1
```

Comment:
minor: move the common parts of the if and else out of the else

Reply:
Yep, you are correct. I'm going to remove this file actually as we have post_install.sh.